POGOLib.Core/Net/RpcClient.cs

```diff
@@ -515,7 +515,7 @@ private Task<ByteString> SendRemoteProcedureCall(RequestEnvelope requestEnvelope
                     while (_rpcQueue.TryDequeue(out processRequestEnvelope))
                     {
 //                        var diff = Math.Max(0, DateTime.Now.Millisecond - LastRpcRequest.Millisecond);
-                        var diff = Math.Min((int)(DateTime.UtcNow - LastRpcRequest.ToUniversalTime()).TotalMilliseconds, Configuration.ThrottleDifference);
+                        var diff = (int)Math.Min((DateTime.UtcNow - LastRpcRequest.ToUniversalTime()).TotalMilliseconds, Configuration.ThrottleDifference);
```

Comment:
maybe 
```
int diff = 0; 

diff = (int)Math.Min((DateTime.UtcNow - LastRpcRequest.ToUniversalTime()).TotalMilliseconds, Configuration.ThrottleDifference);
if (diff < 0) diff = 0;
``` 
idee only

Reply:
Under normal conditions LastRpcRequest always comes before UtcNow => So the difference should always be greater than zero. 
The problem here was that initial LastRpcRequest has a value of '01/01/0001' so the difference between the current date and LastRpcRequest was a big double value, and casting that number to a int resulted in a negative value.